src/VirtoCommerce.ImageToolsModule.Data/Services/DefaultImageService.cs

```diff
@@ -1,26 +1,63 @@
 using System;
 using System.IO;
+using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
 using SixLabors.ImageSharp;
 using SixLabors.ImageSharp.Formats;
 using SixLabors.ImageSharp.Formats.Jpeg;
 using SixLabors.ImageSharp.PixelFormats;
 using VirtoCommerce.AssetsModule.Core.Assets;
+using VirtoCommerce.ImageToolsModule.Core;
 using VirtoCommerce.ImageToolsModule.Core.Models;
 using VirtoCommerce.ImageToolsModule.Core.Services;
+using VirtoCommerce.Platform.Core.Settings;
 
 namespace VirtoCommerce.ImageToolsModule.Data.Services
 {
     public class DefaultImageService : IImageService
     {
         private readonly IBlobStorageProvider _storageProvider;
         private readonly ILogger<DefaultImageService> _logger;
+        private readonly ISettingsManager _settingsManager;
+        private IImageFormat[] _allowedImageFormats;
 
-        public DefaultImageService(IBlobStorageProvider storageProvider, ILogger<DefaultImageService> logger)
+        public DefaultImageService(IBlobStorageProvider storageProvider, ISettingsManager settingsManager, ILogger<DefaultImageService> logger)
         {
             _storageProvider = storageProvider;
             _logger = logger;
+            _settingsManager = settingsManager;
```

Comment:
It is more pleasant and understandable when the order of the constructor parameters coincides with the order of their assignment.